src/data/legacyModuleDescriptions.json

```diff
@@ -264,6 +264,6 @@
 	"extendedDescription": "Partial localisation. Does not translate the whole site.\nFor Japanese, you may also want to change your title language at https://anilist.co/settings/media\n\nWant to contribute? See: https://github.com/hohMiyazawa/Automail/issues/69",
 	"type": "select",
 	"categories": ["Script","Newly Added"],
-	"values": ["English","Norsk","Português","Deutsch","日本語","Åarjelsaemie","Svenska","English (US)"]
+	"values": ["English","Norsk","Português","Deutsch","日本語","Åarjelsaemie","Svenska","English (US)","Spanish"]
```

Comment:
Use "Español" here, and move it between "Norsk" and "Português" to keep it sorted by completeness. (To make it more complete than Norwegian too, you would have to add staff credit translations, see "special_keys.txt" and the bottom half of "Norwegian.json" for that)

Reply:
To be clear, credit translations are completely optional, and you don't have to do that for this pull request to be merged.

Reply:
Credit translations added for a better experience :D

Reply:
Ordered